ihmc-avatar-interfaces/src/main/java/us/ihmc/avatar/colorVision/BlackflyImagePublisher.java

```diff
@@ -120,7 +121,8 @@ private void publishDistortedColor(RawImage distortedImageToPublish)
          distortedImageMessage.getOrientation().set(imageToPublish.getOrientation());
          distortedImageMessage.setSequenceNumber(imageToPublish.getSequenceNumber());
          CameraModel.EQUIDISTANT_FISHEYE.packMessageFormat(distortedImageMessage);
-         ImageMessageFormat.COLOR_JPEG_BGR8.packMessageFormat(distortedImageMessage);
+         PixelFormat.BGR8.packImageMessage(distortedImageMessage);
+         CompressionType.NVJPEG.packImageMessage(distortedImageMessage);
          ros2DistoredImagePublisher.publish(distortedImageMessage);
```

Comment:
Agreed. This could be the clearest way written:
```
distortedImageMessage.setCameraModel(CameraModel.EQUIDISTANT_FISHEYE.toByte());
distortedImageMessage.setPixelFormat(PixelFormat.BGR8.toByte());
distortedImageMessage.setCompressionType(CompressionType.NVJPEG.toByte());
```